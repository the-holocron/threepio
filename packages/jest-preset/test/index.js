const config = require("../jest-preset.json");

describe("preset", () => {
	describe("jest", () => {
		test("should render the configuration", () => expect(config).toMatchSnapshot());
	});
});
