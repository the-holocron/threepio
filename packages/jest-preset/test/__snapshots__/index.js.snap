// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preset Jest should render the configuration 1`] = `
Object {
  "collectCoverage": true,
  "collectCoverageFrom": Array [
    "<rootDir>/src/**/*.{js,jsx}",
    "!<rootDir>/**/{__mocks__, mocks}/**",
    "!<rootDir>/**/*.{mocks,mock}.js",
    "!<rootDir>/**/{__stories__,stories,story}/**",
    "!<rootDir>/**/{__tests__,tests,test}/__snapshots__/**",
  ],
  "coverageThreshold": Object {
    "global": Object {
      "branches": 80,
      "functions": 80,
      "lines": 80,
      "statements": 80,
    },
  },
  "displayName": "Unit Tests",
  "modulePathIgnorePatterns": Array [
    "<rootDir>/dist/",
  ],
  "testPathIgnorePatterns": Array [
    "/coverage/",
    "/node_modules/",
    "((\\\\.|/)(mocks|mock))\\\\.[jt]sx?$",
  ],
  "testRegex": Array [
    "(/__tests__/.*|(\\\\.|/)(test|spec))\\\\.[jt]sx?$",
    "(/tests?/.*|(\\\\.|/)(test|spec))\\\\.[jt]sx?$",
  ],
  "transform": Object {
    "^.+\\\\.js$": "babel-jest",
  },
  "transformIgnorePatterns": Array [
    "<rootDir>/node_modules/",
  ],
}
`;
